test/plug/telemetry_test.exs

```diff
@@ -15,6 +15,23 @@ defmodule Plug.TelemetryTest do
     end
   end
 
+  defmodule MyRoutePlug do
+    use Plug.Router
+
+    plug :match
+    plug :dispatch
+
+    def call(conn, opts) do
+      super(conn, opts)
+    after
+      Process.put(:plug_forward_call, true)
+    end
+
+    get "/" do
+      resp(conn, 200, "Response")
+    end
+  end
```

Comment:
We should move this plug/router_test.exs instead. :)

Reply:
Done.